### PR TITLE
Add extra permissions for azure ns to ci-operator

### PR DESCRIPTION
**Pull request by 0xmichalis**
@stevekuznetsov the other option I guess is to move our base build in the ci namespace. Thoughts?

### CODE REVIEW

**Comment by kargakis** [comment]
@openshift/sig-azure

File: projects/azure/rbac.yaml [hunk 1]
```diff
@@ -75,3 +75,69 @@ items:
...
+    name: system:unauthenticated
+  roleRef:
+    kind: Role
+    apiGroup: rbac.authorization.k8s.io
+    name: anonymous-image-access
+    namespace: azure
+- kind: Role
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: anonymous-image-access
```
Comment (jim-minter) [inline projects/azure/rbac.yaml]: think I'd call this image-access ?

**Comment by jim-minter** [comment]
lgtm, @kargakis you tag and merge when you're ready

File: projects/azure/rbac.yaml (hunk 1 above)
Comment (kargakis) [reply projects/azure/rbac.yaml]: yeah, fixed

**Pull request merged**